test/performance/workloads/performance-test-config.yaml

```diff
@@ -41,6 +41,12 @@ tuningSets:
   parallelismLimitedLoad:
     parallelismLimit: 100
 
+# PoissonLoad defines a load with with Poisson arrival time
+# for a given rate parameter λ (ExpectedActionsPerSecond)
```

Comment:
I would change the order 
for a given rate parameter ExpectedActionsPerSecond (λ in the Poisson model)

Reply:
Fixed! Thanks